lib/global-styles.php

```diff
@@ -312,6 +373,28 @@ function gutenberg_experimental_global_styles_get_support_keys() {
 	);
 }
 
+/**
+ * Returns how the presets css variables are structured on the global styles data.
+ *
+ * @return array Presets structure
+ */
+function gutenberg_experimental_global_styles_get_presets_structure() {
```

Comment:
For keeping things consistent, can this be modeled as a simple array?

```php
return array(
    'color'     => array( 'color', 'palette' ),
    'gradient'  => array( 'color', 'gradients' ),
    'font-size' => array( 'typography', 'fontSizes' ),
);
```

I'd also suggest that we rename `font-size` to `fontSize`. It's best to group the shape changes in one go, as to minimize disruption for consumers (we don't want to change the shape in every release).

Reply:
'font-size' is used  to generate the present variable name that should contain font-size and not 'fontSize'. This mapping is only to generate preset variable names so I guess font-size makes sense otherwise we would need yet another mapping.

Reply:
Can't we reuse the existing `gutenberg_experimental_global_styles_get_css_property`?